docs/sql-manual/sql-statements/job/CREATE-JOB.md

```diff
@@ -25,147 +25,137 @@ specific language governing permissions and limitations
-
-There are two types of Jobs: ONE_TIME and RECURRING. The ONE_TIME type of Job triggers at a specified time point and is mainly used for one-time tasks, while the RECURRING type of Job triggers at specified time intervals and is used for periodically recurring tasks. The RECURRING type of Job can specify a start time and an end time using STARTS\ENDS. If the start time is not specified, the first execution time is set to the current time plus one scheduling period. If the end time is specified and the task completes execution by reaching the end time (or exceeds it, or the next execution period exceeds the end time), the Job status is updated to FINISHED, and no more Tasks are generated.
-
-#### Job Status
-
-A Job has four states (RUNNING, STOPPED, PAUSED, FINISHED), with the initial state being RUNNING. A Job in the RUNNING state generates Tasks based on the specified scheduling period. When a Job completes execution and reaches the end time, the status changes to FINISHED.
+Doris Job is a task that runs according to a set plan. It is used to trigger predefined operations at a specific time or a specified time interval, so as to help us automatically perform some tasks. Functionally, it is similar to the scheduled task on the operating system (such as cron in Linux and scheduled tasks in Windows).
 
-A Job in the RUNNING state can be paused, which means it will no longer generate Tasks.
+There are two types of jobs: `ONE_TIME` and `RECURRING`. Among them, the `ONE_TIME` type of job will be triggered at a specified time point. It is mainly used for one-time tasks, while the `RECURRING` type of job will be triggered cyclically within a specified time interval. This method is mainly used for periodic tasks.
+The `RECURRING` type of job can specify the start time and end time, that is, `STARTS\ENDS`. If the start time is not specified, the default first execution time is the current time + one scheduling cycle. If the end time is specified, the task execution is completed. If the end time is reached (or exceeded, or the next execution cycle will exceed the end time), it will be updated to the FINISHED state, and no more tasks will be generated at this time.
 
-A Job in the PAUSED state can be resumed by performing the RESUME operation, changing the state to RUNNING.
+There are 4 states for a job (`RUNNING`, `STOPPED`, `PAUSED`, `FINISHED`). The initial state is RUNNING. A job in the RUNNING state will generate a TASK for execution according to the established scheduling cycle. When the job is completed and reaches the end time, the state changes to `FINISHED`.
 
-A Job in the STOPPED state is triggered by the user, which cancels the running Job and then deletes it.
+A job in the RUNNING state can be paused, that is, it will be paused, and no more tasks will be generated.
```

Comment:
这一段话里面的 state 都加上反引号，更好一点？
```suggestion
A job in the `RUNNING` state can be paused, that is, it will be paused, and no more tasks will be generated.
```